modules/contact/test/tests/bouncing-block-contact/mixed-weighted-gap-swapped.i

```diff
@@ -0,0 +1,154 @@
+starting_point = 2e-1
+
+# We offset slightly so we avoid the case where the bottom of the secondary block and the top of the
+# primary block are perfectly vertically aligned which can cause the backtracking line search some
+# issues for a coarse mesh (basic line search handles that fine)
+offset = 1e-2
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+  diffusivity = 1e0
+  correct_edge_dropping = true
+[]
+
+[Mesh]
+  second_order = true
+  [file_mesh]
+    type = FileMeshGenerator
+    file = long-bottom-block-1elem-blocks-coarse.e
+  []
+[]
+
+[Variables]
+  [disp_x]
+    block = '1 2'
+    scaling = 1e1
+    order = SECOND
+  []
+  [disp_y]
+    block = '1 2'
+    scaling = 1e1
+    order = SECOND
+  []
+  [frictional_normal_lm]
+    block = 4
+    scaling = 1e3
+  []
+[]
+
+[ICs]
+  [disp_y]
+    block = 2
+    variable = disp_y
+    value = '${fparse starting_point + offset}'
+    type = ConstantIC
+  []
+[]
+
+[Kernels]
+  [disp_x]
+    type = MatDiffusion
+    variable = disp_x
+  []
+  [disp_y]
+    type = MatDiffusion
+    variable = disp_y
+  []
+[]
+
+[Constraints]
+  [frictional_normal_lm]
+    type = ComputeWeightedGapLMMechanicalContact
+    primary_boundary = 10
+    secondary_boundary = 20
+    primary_subdomain = 3
+    secondary_subdomain = 4
+    variable = frictional_normal_lm
+    disp_x = disp_x
+    disp_y = disp_y
+    normalize_c = true
```

Comment:
I'll try to remember to expose this parameter through the action next time we update it.